src/QueryBuilder/QueryBuilder/helpers/query.test.js

```diff
@@ -122,8 +122,8 @@ describe('mongoQueryToSource()', () => {
         { user_last_name: { $lt: 10 } },
         { user_last_name: { $gte: 'value' } },
         { languages: { $in: ['value', 'value2'] } },
-        { user_full_name: { $regex: '^abc' } },
-        { user_full_name: { $regex: '^abc' } },
+        { user_full_name: { $regex: 'abc' } },
```

Comment:
Should this line be duplicated?

Reply:
`initial` and `source` need to stay in sync. I think you're right that it's a duplicate, @vashjs, but we'd need to purge the related lines throughout the rest of this test file too: lines 62-67, and line 85.

Reply:
@zburke  thanks for response, you are right. I think we will do small tests clenup in feature stories.

Reply:
I removed duplicates. Thanks for your comments.